app/controllers/ems_common.rb

```diff
@@ -464,6 +464,10 @@ def button
       javascript_redirect :controller         => "cloud_volume",
                           :action             => "new",
                           :storage_manager_id => params[:id]
+    elsif params[:pressed] == "cloud_volume_snapshot_create"
+      javascript_redirect :controller => "cloud_volume",
+                          :action     => "snapshot_new",
+                          :id         => find_checked_ids_with_rbac(CloudVolume).first
```

Comment:
Aah, right :) Well, if you *are* fixing this, then yeah, feel free to change the others :).

Basically, going forward, `find_records_with_rbac` should be the only remaining RBAC function, except for places with very specific needs (like `Rbac.filtered` when you already have a AR query object, or cases when you only ever need ids not records, etc.).